mylyn.commons/org.eclipse.mylyn.commons.repositories.ui/src/org/eclipse/mylyn/commons/repositories/ui/RepositoryLocationPart.java

```diff
@@ -118,17 +118,6 @@ public void modifyText(ModifyEvent event) {
 			apply();
 		}
 
-		@SuppressWarnings("unused")
```

Comment:
This is weird, needs closer look.
Perhaps we need to call this method during `updateProxyEnablement` execution

Reply:
Should we push this one?

Reply:
Merging.